test/cases/build-image.sh

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bash
+#
+# Build an image and store the hash of the manifest in S3 if successful.
+
+set -euxo pipefail
+
+distro="${1}"
+imgtype="${2}"
+config="${3}"
+
+echo "Installing dependencies"
+sudo dnf install -y go gpgme-devel gcc osbuild osbuild-luks2 osbuild-lvm2 osbuild-ostree osbuild-selinux
+
+echo "Building image ${distro}/${imgtype} using config ${config}"
+cat "${config}"  # print the config for logging
+sudo go run ./cmd/build -output ./build -distro "${distro}" -image "${imgtype}" -config "${config}"
+
+echo "Build finished!!"
+
+echo "Registering successful build in S3"
+
+config_name=$(jq -r .name "${config}")
+manifest=(./build/*/manifest.json)  # there should only be one
+manifest_hash=$(sha256sum "${manifest[0]}" | awk '{ print $1 }')
+
+arch=$(uname -m)
+
+# TODO: Include osbuild commit hash
```

Comment:
For now you could include `osbuild --version` first so any large discrepancies show up.

Reply:
Done!